<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VSComputerForm</class>
 <widget class="QDialog" name="VSComputerForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>310</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>310</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>310</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>91</y>
     <width>130</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Computer
Level</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>90</y>
     <width>401</width>
     <height>71</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="easy_button">
      <property name="minimumSize">
       <size>
        <width>60</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>120</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>.AppleSystemUIFont</family>
        <pointsize>20</pointsize>
        <strikeout>false</strikeout>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: qlineargradient(spread:pad, x1:0.152183, y1:0.00568182, x2:0.816754, y2:1, stop:0 rgba(0, 152, 0, 255), stop:1 rgba(43, 255, 46, 255));
border-radius: 8px;
color: black</string>
      </property>
      <property name="text">
       <string>Easy</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="medium_button">
      <property name="minimumSize">
       <size>
        <width>60</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>120</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>.AppleSystemUIFont</family>
        <pointsize>20</pointsize>
        <strikeout>false</strikeout>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: qlineargradient(spread:pad, x1:0.126, y1:0, x2:0.790576, y2:1, stop:0.272251 rgba(255, 255, 94, 255), stop:1 rgba(179, 184, 0, 255));
color: black;
border-radius: 8px;</string>
      </property>
      <property name="text">
       <string>Medium</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="hard_button">
      <property name="minimumSize">
       <size>
        <width>60</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>120</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>.AppleSystemUIFont</family>
        <pointsize>20</pointsize>
        <weight>50</weight>
        <bold>false</bold>
        <strikeout>false</strikeout>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: qlineargradient(spread:pad, x1:0.131, y1:0, x2:0.853403, y2:1, stop:0 rgba(255, 0, 0, 255), stop:1 rgba(79, 0, 0, 255));
border-radius: 8px;
color: black;</string>
      </property>
      <property name="text">
       <string>Hard</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>310</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Want to make first move?</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>220</y>
     <width>21</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>220</y>
     <width>41</width>
     <height>25</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
   <property name="keyboardTracking">
    <bool>false</bool>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>6</number>
   </property>
   <property name="value">
    <number>4</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>200</y>
     <width>101</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Count of
stones</string>
   </property>
  </widget>
  <zorder>label_2</zorder>
  <zorder>label</zorder>
  <zorder>horizontalLayoutWidget</zorder>
  <zorder>label_3</zorder>
  <zorder>checkBox</zorder>
  <zorder>spinBox</zorder>
  <zorder>label_4</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
